.kokoro/tests/run_tests.sh

```diff
@@ -154,6 +154,20 @@ testing/btlr "${btlr_args[@]}"
 RTN=$?
 cd "$ROOT"
 
+# Setup DRIFT region tag parser
+# (only run on *some* builds)
+if [ "${INJECT_REGION_TAGS:-}" == "true" ]]; then
+    # install PyYaml (used by the DRIFT region tag parsing system)
```

Comment:
It's not obvious from the log, so can you add info log?

```
    echo "Downloading region tag parser"
```

or something?

Also, do you prefer always installing it from HEAD?

If you install with a SHA1 hash or release tag, can you also log the hash or the release tag?

Reply:
1) Fixed missing bracket
2) Added debug log statements
3) Yes - installing from `HEAD` is the intent here (esp. since we don't need to use a commit hash to bust Docker caches!)

Reply:
Ok, this raises another question. How is the parser script tested?

Reply:
The _Python_ script itself is fetched [from this repo](https://github.com/googlecloudplatform/repo-automation-playground/tree/master/wizard-py); there are tests in that folder.

Reply:
Do you have CI builds setup?

Reply:
Not yet - though we plan to add them in the coming months.

(If you'd prefer to do that immediately after merging this PR though, that can be done.)